special-interest-groups/sig-template/roles-and-organization-management.md

```diff
@@ -0,0 +1,73 @@
+# Template SIG Roles and Organization Management
+
+<!--
+Outline the organism specific to this SIG, as well as those that differ from [SIG Governance](sig-governance.md)
+-->
+
+## Join and Promotion
+
+We mainly follow the instructions in [TiDB Developer Group](../../architecture/README.md#tidb-developer-group).
+It lists common requirements, responsibilities and privileges of each role in developer group.
+
+### Join
+
+You are welcomed to be a [contributor](../../architecture/README.md#contributor) once one of your PR about `${module name}` is merged.
+To have a discusstion with us, just join our slack channel **#sig-${name}** in [tidbcommnity](https://pingcap.com/tidbslack) slack workspace.
+
+### Promotion
+
+#### Be Promoted to [`Active Contributor`](../../architecture/README.md#active-contributor)
+
+SIG Specific Requirements:
+
+* Joined the sig-${name} slack channel.
+
+#### Be promoted to [`Reviewer`](../../architecture/README.md#reviewer)
+
+SIG Specific Requirements:
+
+* Have at least  **${a certain number}** PRs merged in the `${module name}` package within one year.
+* Fix at least **${a certain number}** issues whose difficulty is medium or above.
+* Presented design ideas and adopted them in more than  **${a certain number}** executable task.
+
+
+#### Be promoted to [`Committer`](../../architecture/README.md#committer)
+
+SIG Specific Requirements:
+
+* Exhibited good technical judgment.
+* Fix at least  **${a certain number}** issues whose difficulty is hard.
+* Have reviewed at least  **${a certain number}** PRs in the `${module name}` package within one year.
+* Play an important role in a big feature.
+
+## Quit and Demotion
+
+### Active Contributor
+
+**Automatically retired**:
+
+* No contribution to the `${module name}` package within one year.
+
+**How to rejoin**:
+
+* Should meet the requirements for Active Contributor again.
+
+### Reviewer
+
+**Automatically demoted**:
+
+For a Reviewer to be demoted to an Active Contributor, he/she must satisfy at
+least 1 of the following conditions:
+
+* Haven't reviewed or merged any PR in the `${module name}` package within 3 months.
+* Haven't appeared in the SIG meeting without any reason for 3 times continuously.
+* At least 2 Committers or Maintainers agree that the Reviewer is not qualified.
+
+**How to retain**:
+
+* Have reviewed or merged  **${a certain number}** PRs within 1 month.
+
+### Voluntarily Quit or Demote
+
+Please email to [Yanqing](mailto:zhangyanqing@pingcap.com) if you are
```

Comment:
```suggestion
Please create a PR to  update  remove yourself from the `member-list.md` if you are
```

Reply:
I think mailing to Yanqing is very informal and totally unnecessary. Opening a PR and letting Tech Lead or committer review makes sense.

Reply:
This comment is not resolved yet~ @Soline324 @winkyao

Reply:
agreed.